### DISCUSSION TITLE
USB to UART (VCP) Experiences with the UV-K5 family

### DISCUSSION THREAD

**Discussion by ludwich66** [ludwich66/Quansheng_UV-K5_Wiki#11] [Show and tell]
Post your experience to feed the table in the wiki!
https://github.com/ludwich66/Quansheng_UV-K5_Wiki/wiki/Programming-Cable

**Comment by manujedi** [comment]
On Linux:

- CH341 works with [CHIRP](https://chirp.danplanet.com/projects/chirp/wiki/Home) and [k5prog](https://github.com/sq5bpf/k5prog)
  - for me, it does not work with the official software with wine. But that may be a wine problem
- Picoprobe [Appendix A: Using Picoprobe -> Using Picoprobe's UART](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) works with [CHIRP](https://chirp.danplanet.com/projects/chirp/wiki/Home) and [k5prog](https://github.com/sq5bpf/k5prog)
  - same results on wine
